src/main/scala/org/camunda/feel/impl/builtin/ListBuiltinFunctions.scala

```diff
@@ -518,4 +519,16 @@ class ListBuiltinFunctions(private val valueMapper: ValueMapper) {
       }
     )
 
+  private def partitionFunction =
+    builtinFunction(
+      params = List("list", "size"),
+      invoke = { case List(ValList(list), ValNumber(size)) =>
+        if (size.intValue > 0) {
+          ValList(list.grouped(size.intValue).map(l => ValList(l)).toList)
+        } else {
+          ValNull
```

Comment:
:wrench: We could return an error to explain why the result is `null`.

```suggestion
          ValError(s"'size' should be greater than zero but was '$size'")
```

Refer to the [guidelines](https://github.com/camunda/feel-scala/blob/main/CONTRIBUTING.md#adding-a-new-built-in-function).